modules/common/condition/funcs_test.go

```diff
@@ -422,6 +422,45 @@ func TestMirror(t *testing.T) {
 	g.Expect(targetCondition.Message).To(BeIdenticalTo(trueReady.Message))
 }
 
+func TestMirrorReadyTrueAndFalse(t *testing.T) {
```

Comment:
I am not against adding this, but if we remove line 415 `conditions.MarkTrue(trueB.Type, trueB.Message)` it would be basically this test. then there would be `falseBError` and `ReadyCondition` is True

Reply:
True, I can do that.